domains/panel.sono.json

```diff
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "username": "getSono",
+    "email": "sono.arbeit@gmail.com"
+  },
+  "record": {
+    "A": [
+      "104.28.224.241"
```

Comment:
I cannot reach your website, keeps on loading

Reply:
or show a preview of it

Reply:
german internet.